R/plot_spawning_biomass.R

```diff
@@ -126,5 +137,23 @@ plot_spawning_biomass <- function(
     )
 
   plt_fin <- suppressWarnings(add_theme(plt))
+
+  # add alt text and caption
+  plt_fin <- plt_fin +
+    ggplot2::labs(caption = caps_alttext[[1]],
+                  alt = caps_alttext[[2]]
+    )
+
+  # export figure to rda if argument = T
+  if (make_rda == TRUE){
```

Comment:
In it's current state, the function exports the rda while also plotting the plot. Was this intentional? How do you think this would fit in with assessment scientists' workflows? Either for report writing or general analyses

Reply:
if the scientist is just interested in looking at the plot, or saving it as a basic image file, they can run it with make_rda = F. From there, they can save it as an object within their environment and save it as an image file to their local machine if they'd like. They can also alter it by adding commands (e.g., `+ theme_bw()`) or modifying it (e.g., with `ggplot_build()`), and then save that modified object. If they want to save the standard plot, its alt text, and its caption as an rda, then they can rerun the basic plot function with make_rda = T. We could provide a vignette or few lines of code showing them how to save an altered plot as an rda list.